src/main/java/uk/gov/pay/directdebit/payments/api/PaymentRequestValidator.java

```diff
@@ -15,7 +15,7 @@
     private final static String REFERENCE_KEY = "reference";
 
     private final static int MIN_AMOUNT = 1;
-    private final static int MAX_AMOUNT = 10000000;
+    private final static int MAX_AMOUNT = 500000;
```

Comment:
I’m going to be really picky here and say this would be clearer with underscores e.g. `5000_00` (yes, you can do that in Java!).

Also, if it’s not too much trouble could you rename this constant to be `MAX_AMOUNT_IN_PENCE`? It’s always nice when variable names describe their units (it would be even nicer to use a specific type but let’s not go there right now).